backend/gateway/core.go

```diff
@@ -72,6 +73,7 @@ var Modules = module.Factory{
 	slackbotmod.Name:           slackbotmod.New,
 	sourcecontrol.Name:         sourcecontrol.New,
 	topologymod.Name:           topologymod.New,
+	proxymod.Name:              proxymod.New,
```

Comment:
nit: can you alpha this list

Reply:
yeah sure

Reply:
sorted all the resolvers, services & mods while i was there.